src/tfont/converters/ufoConverter.py

```diff
@@ -70,17 +73,18 @@ def open(self, path, font=None):
             master.name = info.styleName
         for blues in (info.postscriptBlueValues, info.postscriptOtherBlues):
             for yMin, yMax in zip(blues[::2], blues[1::2]):
-                master.alignmentZones.append(AlignmentZone(yMin, yMax-yMin))
+                master.alignmentZones.append(AlignmentZone(yMin, yMax - yMin))
         if info.postscriptStemSnapH:
             master.hStems = info.postscriptStemSnapH
         if info.postscriptStemSnapV:
             master.vStems = info.postscriptStemSnapV
-        for g in ufo.guidelines:
-            guideline = Guideline(
-                x=g.x or 0, y=g.y or 0, angle=g.angle or 0, name=g.name or ""
-            )
-            # ufo guideline color and identifier are skipped
-            master.guidelines.append(guideline)
+        if ufo.guidelines:
```

Comment:
@anthrotype Can you make this a list?

Reply:
I could, but I thought that since everything in fontinfo.plist is optional, including fontinfo.plist file itself, all the list-like attributes should default to None just like the rest of the attributes.

Reply:
Otherwise you end up with a spurious fontinfo.plist that only has a bunch of empty guidelines or similar plist empt arrays

Reply:
Alright, we have our own format now so let's take it the way it is

Reply:
Maybe I could have Font.guidelines property getter return an empty list if info.guidelines (which is where they are actually stored) is None.

Reply:
https://github.com/fonttools/ufoLib2/commit/06872e3922b4bd9e5d5e57fc33c41fb20f120043